.changes/unreleased/Under the Hood-20230105-182038.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
+body: Use CircleCI Context for AWS access key environment variables
+time: 2023-01-05T18:20:38.48148-07:00
+custom:
+  Author: dbeatty10
+  Issue: "581"
+  PR: "581"
```

Comment:
This is a no changelog kind of change.  CI changes don't really belong in the changelog.  `Under the Hood` is more of a code change that users shouldn't really notice, such as renaming classes.

Reply:
Cool, I removed the file for changie.

Are there any other types of changes besides CI that we don't add to the changelog?